packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -429,18 +427,16 @@ export function makeRumPublicApi(
       strategy.init(initConfiguration, rumPublicApi)
 
       // Add experimental features here
-      if (isExperimentalFeatureEnabled(ExperimentalFeature.UPDATE_VIEW_NAME)) {
-        /**
-         * Update View Name.
-         *
-         * Enable to manually change the name of the current view.
-         * @param name name of the view
-         * See [Override default RUM view names](https://docs.datadoghq.com/real_user_monitoring/browser/advanced_configuration/#override-default-rum-view-names) for further information.
-         */
-        ;(rumPublicApi as any).updateViewName = monitor((name: string) => {
-          strategy.updateViewName(name)
-        })
-      }
+      /**
+       * Update View Name.
+       *
+       * Enable to manually change the name of the current view.
+       * @param name name of the view
+       * See [Override default RUM view names](https://docs.datadoghq.com/real_user_monitoring/browser/advanced_configuration/#override-default-rum-view-names) for further information.
+       */
+      ;(rumPublicApi as any).setViewName = monitor((name: string) => {
+        strategy.setViewName(name)
+      })
```

Comment:
**💬 suggestion:**
* move this method declaration at the same place as other methods, maybe before [`setViewContext`](https://github.com/DataDog/browser-sdk/blob/eb9055208285b9e3552848115e237b7f588fa700/packages/rum-core/src/boot/rumPublicApi.ts#L446)
* add its type and jsdoc in the `interface RumPublicApi` above